staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -112,6 +112,8 @@ func NewAdmissionController(
 	restMapper meta.RESTMapper,
 	dynamicClient dynamic.Interface,
 ) CELPolicyEvaluator {
+	var _ = informerFactory.Core().V1().ConfigMaps().Informer()
```

Comment:
This line is needed to prevent integration tests from failing for now. Since we dynamically want to look up the informers we run into an issue with how `SharedInformerFactory.Start` only `Run`s the informers which were requested before calling `Start`. 

We could call `Start` ourselves, but then the matter of which `stopCh` to use is a difficult question:
1. If we provide `stopCh` lasting only as long as our dependency of the param type, that design might stop the informer out from under other parts of k8s code used if they dynamically used the same informer once we no longer had need of it and closed the `stopCh`.

2. So, then the only other solution I easily think of is to provide a `stopCh` running as long as the controller/application. That poses a performance problem - possibly keeping around unnecessary informers if a policy started using a type then changed to use a different paramKind. We would not want to keep around the informer.

Any advice @jpbetz @jiahuif ?

Reply:
per offline discussion with jpbetz, since the informer factory does not support our dynamic usage, will change to creating a non-shared informer. At least it is typed.

Reply:
Where you able to resolve @alexzielenski? Per offline conversation it sounded like you were considering creating a new factory to get around the ownership/lifecycle limitations of shared informers?

Reply:
yes that is current state of PR. looks like only one integration test failing now (`kind` key not found). I think may need help from @jiahuif to resolve it:

```
    validatingadmissionpolicy_test.go:706: timed out waiting: namespaces "configmap-v8kbp" is forbidden: ValidatingAdmissionPolicy 'allowed-prefixes' with binding 'allowed-prefixes-binding' denied request: expression 'object.metadata.name.startsWith(params.kind.lowerAscii())' resulted in error: no such key: kind
    validatingadmissionpolicy_test.go:716: namespaces "secret-mnfmk" is forbidden: ValidatingAdmissionPolicy 'allowed-prefixes' with binding 'allowed-prefixes-binding' denied request: expression 'object.metadata.name.startsWith(params.kind.lowerAscii())' resulted in error: no such key: kind
```